released/packages/coq-waterproof/coq-waterproof.1.2.4/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Waterproof library"
+description: """
+  The Waterproof library provides tactics, notations, and mathematical theories geared towards use in Mathematics educational environments. It aims to provide syntax such that proof scripts mimic handwritten mathematical proofs.
+"""
+
+homepage: "https://github.com/impermeable/coq-waterproof"
+dev-repo: "git+https://github.com/impermeable/coq-waterproof.git"
+bug-reports: "https://github.com/impermeable/coq-waterproof/issues"
+maintainer: "j.w.portegies@tue.nl"
+authors: [
+  "Jelle Wemmenhove"
+  "Cosmin Manea"
+  "Lulof Pirée"
+  "Adrian Vrămuleţ"
+  "Tudor Voicu"
+  "Jim Portegies"
+]
+license: "LGPL 3.0"
```

Comment:
```suggestion
license: "LGPL-3.0-only"
```
We try to use [SPDX identifers](https://spdx.org/licenses/) here. Is this LGPL-3.0-only or LGPL-3.0-or-later? In the absence of other information, I'd have to go with the former.